lib/ts/framework/constants.ts

```diff
@@ -13,3 +13,6 @@
  * under the License.
  */
 export const COOKIE_HEADER = "Set-Cookie";
+
+// Define error message for brotli decompression not being supported
+export const BROTLI_DECOMPRESSION_ERROR_MESSAGE = "Brotli decompression not implement, use a middleware instead";
```

Comment:
Please add a middleware that handles decompression before the SuperTokens middleware.